fla/ops/mesa_net/decoding_one_step.py

```
@@ -0,0 +1,149 @@
+    diag_mask = tl.arange(0, BK)[:, None] == tl.arange(0, BK)[None, :]
+    diag_mask = diag_mask & mask_kk
+    b_h_kk_diag = tl.sum(tl.where(diag_mask, b_h_kk, 0.0), axis=1)
+
+    b_x = b_q / (b_h_kk_diag + b_lamb + 1e-5)
+    b_Hx = tl.sum(b_h_kk * b_x[:, None], axis=0)
+    b_r = b_q - b_Hx - b_lamb * b_x
+    b_p = tl.zeros([BK,], dtype=tl.float32)
+    b_p += b_r
+    delta_old = tl.sum(b_r * b_r)
+
+    for i_iter in range(MAX_CG_STEP):
```

Ruff (0.11.9)

Loop control variable `i_iter` not used within loop body

Rename unused `i_iter` to `_i_iter`

(B007)